### DISCUSSION TITLE
articles/old-path-white-clouds

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#139] [Comments]
https://hutusi.com/articles/old-path-white-clouds 

 首先澄清，我非佛教徒，也不信仰任何宗教。读《佛陀传》缘起是在一本书上看到一行禅师的名句“没有通向幸福的路，幸福本身就是路”。上月一行禅师去世，有网友组织了读一行禅师的读书会活动，因此选择了他的《佛陀传》来阅读。本文第一部分“佛陀的经历”和“佛教讲了什么”并非完全遵照该书内容，而是通过网上资料补充而来。

**Comment by xuling1979** [comment]
正念。

**Comment by hutusi** [comment]
> @xuling1979
> 正念。

正念在白领中很流行的样子~~

**Comment by surmon-china** [comment]
归纳地很完整，想点个赞没有按钮。

其中 “并补充完善了八正道等禅修方式” 中的 “禅修方式” 应该是 “修行方式” 更准确，“禅修” 一般指冥想和正念，类似于八正道中的 “正定”，它不是一个包含的关系。

**Reply by hutusi** [reply]
谢谢，确实“禅修”只代表了一种修行方式，已 fix~~

**Comment by fuadam1982** [comment]
色 -> 肉体，那色即是空里面的「色」呢？看英文翻译心经中色 -> form，这个就很合理。而且佛教是有哲学方法论的，讨论的问题也不会这么世俗吧

**Reply by hutusi** [reply]
“色”在五蕴里理解为“肉体”，我觉得跟心经中的色即是空还不完全一样，关于心经的解读可参见另一篇博文《心经说了什么》https://hutusi.com/articles/what-does-the-heart-sutra-say

五蕴的“色”即是肉体重身及所带来的负担，人生所必须要承受的苦。

**Reply by fuadam1982** [reply]
这个解释也是合理的，只是看佛法还是要参考原文和英文翻译。中文是high-context的语言一词多义一句多义。让读者自行领悟而不是将清原意后再根据自身进行扩展，阅读和理解起来就太费劲

**Reply by hutusi** [reply]
是的，特别是文言文翻译的，没有英文那么直白。
